cmn/daq_common.h

```diff
@@ -8,7 +8,7 @@ typedef int32_t daq_DamperPos_t;
 typedef int32_t daq_GearPos_t;
 typedef int32_t daq_SteeringWhlPos_t;
 typedef int32_t daq_Strain_t;
-typedef int32_t daq_BatteryV_t;
+typedef float daq_BatteryV_t;
```

Comment:
Sorry could you change this uint32_t as an easier unit to handle for communications